pyproject.toml

```diff
@@ -50,8 +50,9 @@ msal = "^1.32.3"
 [tool.poetry.group.dev.dependencies]
 black = "^24.2.0"
 pytest = "^8.0.2"
-pytest-asyncio = "^0.23.5"
 pytest-dotenv = "^0.5.2"
+pytest-anyio = "^0.0.0"
```

Comment:
@benjigifford I believe that's wrong package, you probably want to use this one: https://anyio.readthedocs.io/en/1.4.0/basics.html (I'm not sure why we need it)